MediaProcessor/CMakeLists.txt

```diff
@@ -10,21 +10,23 @@ if(NOT CMAKE_BUILD_TYPE)
   set(CMAKE_BUILD_TYPE Release)
 endif()
 
-include_directories(include)
-include_directories(${CMAKE_SOURCE_DIR}/third_party/nlohmann)
+# Fetch Google Test
+include(FetchContent)
 
+# Set the policy to NEW
+cmake_policy(SET CMP0135 NEW)
```

Comment:
This set fetch content policy to new. This policy is introduced in CMake 3.19. I use it in case It didn't fetch GTest. But i removed it in future commits. ( yet to be pushed )